setup-ovs.sh

```diff
@@ -39,38 +54,33 @@ if [[ $(nmcli conn | grep -c ovs) -eq 0 ]]; then
 
   mac=$(sudo nmcli -g GENERAL.HWADDR dev show $default_device | sed -e 's/\\//g')
 
-  # delete old bridge if it exists
-  ovs-vsctl --if-exists del-br brcnv
-  
   # make bridge
-  nmcli conn add type ovs-bridge conn.interface brcnv
+  nmcli conn add type ovs-bridge conn.interface brcnv conn.id ovs-bridge-brcnv
```

Comment:
I think `conn.id` is a synonym for `con-name` so let's use either this or that (but not both)

Reply:
So lets stay with con-name ? 
I think its more clear then conn.id 

@RamLavi  @phoracek  Thoughts?

Reply:
Changed it to ```con-name```